composer.json

```
@@ -62,8 +62,8 @@
     },
     "require": {
         "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
-        "oat-sa/generis": ">=15.39.0",
-        "oat-sa/tao-core": ">=54.23.0",
+        "oat-sa/generis": "dev-fix/adf-1828/translation-data-in-datastore as 99.99",
```

- [ ] @shpran Please, revert the generis change

```suggestion
        "oat-sa/generis": ">=15.39.0",
```